Disable building the documentation to work around an issue (segfault?) with
one of the generator executables in the CLANGARM64 environment:
```
  incd="`test -f defsincdate || echo './'`defsincdate"; \
  ./mkdefsinc -C . --date "`cat $incd 2>/dev/null`" \
      gpgme.texi uiserver.texi lesser.texi gpl.texi >defs.inc
  mkdefsinc: taking date from 'gpgme.texi'
  make[2]: *** [Makefile:905: defs.inc] Error 139
```

diff -urN gpgme-1.23.2/Makefile.am.orig gpgme-1.23.2/Makefile.am
--- gpgme-1.23.2/Makefile.am.orig	2023-08-31 11:19:00.000000000 +0200
+++ gpgme-1.23.2/Makefile.am	2024-11-07 09:06:17.328590400 +0100
@@ -48,7 +48,7 @@
 tests =
 endif
 
-SUBDIRS = src ${tests} doc lang
+SUBDIRS = src ${tests} lang
 
 # Fix the version of the spec file.
 dist-hook: gen-ChangeLog
diff -urN gpgme-1.23.2/lang/qt/Makefile.am.orig gpgme-1.23.2/lang/qt/Makefile.am
--- gpgme-1.23.2/lang/qt/Makefile.am.orig	2018-12-03 10:37:25.000000000 +0100
+++ gpgme-1.23.2/lang/qt/Makefile.am	2024-11-07 09:18:47.655681200 +0100
@@ -24,6 +24,6 @@
 tests =
 endif
 
-SUBDIRS = src ${tests} doc
+SUBDIRS = src ${tests}
 
 EXTRA_DIST = README
